resteasy-client-microprofile-base/src/main/java/org/jboss/resteasy/microprofile/client/RestClientBuilderImpl.java

```diff
@@ -527,8 +539,6 @@ private void verifyBeanPathParam(Class<?> beanType, Map<String, Object> paramMap
                         : (ResteasyUriBuilder) new ResteasyUriBuilderImpl().uri(classPathAnno.value() + "/" + methodPathAnno.value());
             } else if (classPathAnno != null) {
                 template = (ResteasyUriBuilder) new ResteasyUriBuilderImpl().uri(classPathAnno.value());
-            } else {
-                template = null;
```

Comment:
That one I don't know if it's something you fixed later or if it's something we changed on purpose.